style.css

```diff
@@ -554,107 +610,274 @@ button:active:enabled {
   margin-left: 10px;
 }
 
-.form-section {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  background-image: url(./images/background/Contactcolor.png);
-  background-repeat: no-repeat;
-  background-position: right;
-  width: 375px;
-  height: 779px;
+.dash1 {
+  border: 1px solid #dfe1e6;
+  border-radius: 10px;
+  background-color: #dfe1e6;
+  width: 303px;
 }
 
-.form-section > h2 {
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  color: #fff;
-  width: 801px;
-  text-align: center;
-  margin-top: 120px;
-  margin-bottom: 20px;
-}
+@media only screen and (min-width: 650px) {
```

Comment:
- [ ] Please make sure to use the proper minimum width which is `768px` instead of `650px`, [as per requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=Desktop%3A%20768px%20or%20wider).